webpack/assets/javascripts/react_app/components/common/forms/OrderableSelect/helpers.js

```diff
@@ -0,0 +1,123 @@
+import React, { useImperativeHandle, useRef } from 'react';
+import { DragSource, DropTarget } from 'react-dnd';
+import PropTypes from 'prop-types';
+import { set } from 'lodash';
+
```

Comment:
can you add unit tests for these functions?

Reply:
I have extracted the most important method and tested that one. You think it's enough? I believe it is as we have integrational test and now we test the two important helpers, only the straight forward ones are not tested.